src/main/webapp/resources/js/components/samples/components/fastqc/FastQCCharts.jsx

```diff
@@ -0,0 +1,80 @@
+import { Col, Image, Row, Typography } from "antd";
+import React, { useState } from "react";
+import { useSelector } from "react-redux";
+import styled from "styled-components";
+
+import { getFastQCImages } from "../../../../apis/files/sequence-files";
+import { grey4 } from "../../../../styles/colors";
```

Comment:
Don't use this any more.  Now in dev I have CSS variable.  I will add a comment lower on how to use it.